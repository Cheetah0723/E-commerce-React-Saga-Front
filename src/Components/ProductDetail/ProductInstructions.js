import React from 'react';
import List from '@material-ui/core/List';
import EachProductInstruction from "./EachProductInstruction";
import { Divider } from '@material-ui/core';

const Detail = {
    title: "Detail",
    body: " Combining our signature seamless knit with superior, sculpting design.• Figure - enhancing contours• Elasticated jacquard waistband• DRY technology• 56 % Nylon, 41 % Polyester, 3 % Elastane• Model is 5'5"
};


const Care = {
    title: "Care",
    body: "Machine wash or hand wash with cold water. Do not dry clean it."
}

const Delivery = {
    title: "Delivery",
    body: "Free shipping for orders over 99AUD. Internatonal shipping available."
}

const ProductInstruction = () => {
    return (
        <List
            style={{ textAlign: "center", alignItems: "center" }}
            component="nav"
            aria-labelledby="nested-list-subheader"
        >
            <Divider />
            <EachProductInstruction props={Detail} />
            <Divider />
            <EachProductInstruction props={Care} />
            <Divider />
            <EachProductInstruction props={Delivery} />
            <Divider/>
        </List>
    )

}

export default ProductInstruction;