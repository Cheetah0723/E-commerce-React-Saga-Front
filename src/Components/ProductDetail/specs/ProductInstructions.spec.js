import React from 'react';
import ProductInstructions from "../ProductInstructions"
import renderer from 'react-test-renderer';


it.skip('renders correctly', () => {
    const Detail = {
        title: "Detail",
        body: " Combining our signature seamless knit with superior, sculpting design.• Figure - enhancing contours• Elasticated jacquard waistband• DRY technology• 56 % Nylon, 41 % Polyester, 3 % Elastane• Model is 5'5"
    };


    const Care = {
        title: "Care",
        body: "Machine wash or hand wash with cold water. Do not dry clean it."
    }

    const Delivery = {
        title: "Delivery",
        body: "Free shipping for orders over 99AUD. Internatonal shipping available."
    }
    const component = renderer.create(<ProductInstructions />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
})