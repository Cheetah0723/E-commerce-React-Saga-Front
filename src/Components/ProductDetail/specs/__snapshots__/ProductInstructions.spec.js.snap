// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  aria-labelledby="nested-list-subheader"
  className="MuiList-root MuiList-padding"
  style={
    Object {
      "alignItems": "center",
      "textAlign": "center",
    }
  }
>
  <hr
    className="MuiDivider-root"
  />
  <div
    className="ProductInstruction"
  >
    <div
      aria-disabled={false}
      className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex={0}
    >
      <div
        className="MuiListItemText-root"
        style={
          Object {
            "textAlign": "center",
          }
        }
      >
        <span
          className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
        >
          Detail
        </span>
      </div>
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
        />
      </svg>
    </div>
  </div>
  <hr
    className="MuiDivider-root"
  />
  <div
    className="ProductInstruction"
  >
    <div
      aria-disabled={false}
      className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex={0}
    >
      <div
        className="MuiListItemText-root"
        style={
          Object {
            "textAlign": "center",
          }
        }
      >
        <span
          className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
        >
          Care
        </span>
      </div>
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
        />
      </svg>
    </div>
  </div>
  <hr
    className="MuiDivider-root"
  />
  <div
    className="ProductInstruction"
  >
    <div
      aria-disabled={false}
      className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-button"
      onBlur={[Function]}
      onClick={[Function]}
      onDragLeave={[Function]}
      onFocus={[Function]}
      onKeyDown={[Function]}
      onKeyUp={[Function]}
      onMouseDown={[Function]}
      onMouseLeave={[Function]}
      onMouseUp={[Function]}
      onTouchEnd={[Function]}
      onTouchMove={[Function]}
      onTouchStart={[Function]}
      role="button"
      tabIndex={0}
    >
      <div
        className="MuiListItemText-root"
        style={
          Object {
            "textAlign": "center",
          }
        }
      >
        <span
          className="MuiTypography-root MuiListItemText-primary MuiTypography-body1 MuiTypography-displayBlock"
        >
          Delivery
        </span>
      </div>
      <svg
        aria-hidden={true}
        className="MuiSvgIcon-root"
        focusable="false"
        viewBox="0 0 24 24"
      >
        <path
          d="M16.59 8.59L12 13.17 7.41 8.59 6 10l6 6 6-6z"
        />
      </svg>
    </div>
  </div>
  <hr
    className="MuiDivider-root"
  />
</nav>
`;
